kotsadm/web/src/components/apps/ClusterNodes.jsx

```diff
@@ -22,6 +21,38 @@ export class ClusterNodes extends Component {
     generateCommandErrMsg: ""
   }
 
+  componentDidMount() {
+    this.getNodeStatus();
+  }
+
+  getNodeStatus = async () => {
+    try {
+      const res = await fetch(`${window.env.API_ENDPOINT}/kurl/nodes`, {
+        headers: {
+          "Authorization": Utilities.getToken(),
+          "Accept": "application/json",
+        },
+        method: "GET",
+      });
+      if (!res.ok) {
+        if (res.status === 401) {
+          Utilities.logoutUser();
+          return;
+        }
+        console.log("failed to get node status list, unexpected status code", res.status);
+        return;
+      }
+      const response = await res.json();
+      this.setState({
+        kurl: response,
```

Comment:
should add `kurl` to `state` as well

Reply:
what do you mean?

Reply:
add something like `kurl: null` here https://github.com/replicatedhq/kots/blob/master/kotsadm/web/src/components/apps/ClusterNodes.jsx#L16